tomls/governance/snapshot-contract-mock.toml

```diff
@@ -0,0 +1,26 @@
+[provision.snapshot_record_mock]
+source = "<%= settings.governance_snapshot_record_package %>"
+
+[invoke.ambassador_council_setSnapshotContract]
```

Comment:
Test upgrades fail on base-goerli and polygon-mumbai with same error.

Is it because we haven't yet deployed (judging by how many txns are run that must be the case)?

```ts
Executing [invoke.ambassador_council_setSnapshotContract]...

Failure on step invoke.ambassador_council_setSnapshotContract
CannonTraceError: transaction reverted in contract ambassador_council.ElectionModuleSatellite: NotCallableInCurrentPeriod()

  CALL ambassador_council.CoreProxy.setSnapshotContract("0x854AeE030eFEB8f9C4c778999174A33921613A4F", "true") => NotCallableInCurrentPeriod() (20,000,000 gas)
    DELEGATECALL ambassador_council.CoreRouter.setSnapshotContract("0x854AeE030eFEB8f9C4c778999174A33921613A4F", "true") => NotCallableInCurrentPeriod() (19,661,239 gas)
      DELEGATECALL ambassador_council.SnapshotVotePowerModule.setSnapshotContract("0x854AeE030eFEB8f9C4c778999174A33921613A4F", "true") => NotCallableInCurrentPeriod() (19,351,078 gas)
```